naga/src/proc/mod.rs

```diff
@@ -49,6 +49,7 @@ impl From<super::StorageFormat> for super::ScalarKind {
             Sf::Rgb10a2Uint => Sk::Uint,
             Sf::Rgb10a2Unorm => Sk::Float,
             Sf::Rg11b10Ufloat => Sk::Float,
+            Sf::R64Uint => Sk::Uint,
```

Comment:
question: Doesn't this conflict with #6451?

Reply:
it does not conflict, but now that its merged, width should be a match statement to return 8 for R64